static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -28,28 +83,68 @@ function getPanelDescription(walkthrough: boolean) {
 }
 
 interface TaskProps {
-  description: string;
-  title: string;
+  hidePanel: () => void;
+  task: OnboardingTask;
   status?: 'waiting' | 'completed';
 }
 
-function Task({title, description, status}: TaskProps) {
+function Task({task, status, hidePanel}: TaskProps) {
+  const organization = useOrganization();
+  const router = useRouter();
+
+  const handleClick = useCallback(
+    (e: React.MouseEvent) => {
+      trackAnalytics('quick_start.task_card_clicked', {
+        organization,
+        todo_id: task.task,
+        todo_title: task.title,
+        action: 'clickthrough',
+      });
+
+      e.stopPropagation();
+
+      if (isDemoWalkthrough()) {
+        DemoWalkthroughStore.activateGuideAnchor(task.task);
+      }
+
+      if (task.actionType === 'external') {
+        window.open(task.location, '_blank');
+      }
+
+      if (task.actionType === 'action') {
+        task.action(router);
+      }
+
+      if (task.actionType === 'app') {
+        // Convert all paths to a location object
+        let to =
+          typeof task.location === 'string' ? {pathname: task.location} : task.location;
+        // Add referrer to all links
+        to = {...to, query: {...to.query, referrer: 'onboarding_task'}};
+
+        navigateTo(to, router);
+      }
+      hidePanel();
+    },
+    [task, organization, router, hidePanel]
+  );
```

Comment:
[All this code was copied ](https://github.com/getsentry/sentry/blob/0c2fe3f5582870a8e4216e2d1bcf33e7541aff81/static/app/components/onboardingWizard/task.tsx#L69-L95)from the old task component.